src/Factories/ProcessFactory.php

```diff
@@ -25,17 +27,15 @@ public function __construct(Config $config)
     }
 
     /**
-     * Creates a Git Commit Process that will run on $file.
+     * Creates a VCS Commit Process that will run on $file.
      * @param File $file File that the process will execute on.
      * @return ChurnProcess
      */
-    public function createGitCommitProcess(File $file): ChurnProcess
+    public function createVcsCommitProcess(File $file): ChurnProcess
```

Comment:
I'd rather have a separate process class for each type of version control. So one for Git, one for Mercurial etc.. Will make it more flexible if this project decides to support multiple VCS.

Reply:
I was planning on doing so. This was a mere proof-of-concept :)

Reply:
Sure, no problem.